static/js/containers/ProfileFormContainer.js

```diff
@@ -177,6 +113,24 @@ class ProfileFormContainer extends React.Component {
     }
   }
 
+  simpleActionHelpers: Function = (): ActionHelpers => {
+    const { dispatch } = this.props;
+    return createSimpleActionHelpers(dispatch, [
+      [setWorkDialogVisibility, "setWorkDialogVisibility"],
+      [setWorkDialogIndex, "setWorkDialogIndex"],
+      [clearProfileEdit, "clearProfileEdit"],
+      [setEducationDialogVisibility, "setEducationDialogVisibility"],
+      [setEducationDialogIndex, "setEducationDialogIndex"],
+      [setEducationDegreeLevel, "setEducationDegreeLevel"],
+      [setUserPageDialogVisibility, "setUserPageDialogVisibility"],
+      [setShowEducationDeleteDialog, "setShowEducationDeleteDialog"],
+      [setShowWorkDeleteDialog, "setShowWorkDeleteDialog"],
+      [setDeletionIndex, "setDeletionIndex"],
+      [setShowWorkDeleteAllDialog, "setShowWorkDeleteAllDialog"],
+      [setShowEducationDeleteAllDialog, "setShowEducationDeleteAllDialog"],
```

Comment:
perhaps a little controversial, but one way to avoid this key-value repetition would be to use `import * as ui_actions from 'path/to/actions'`. might make the most sense to put that in the util file rather than here. that way you could just pass a list of strings, eg: `["setWorkDialogVisibility", "setWorkDialogIndex", "clearProfileEdit"]` and we wouldn't have this slightly awkward list-of-lists-with-repeated-names, nor the large import statement above. yes, that would open up the possibility of passing a function name that doesn't exist in actions/ui, but we could make the list of action names a property of this class and test its validity in a unit test